crates/server/build.rs

```diff
@@ -1,12 +1,95 @@
-use std::process::Command;
+use std::path::{Path, PathBuf};
+use std::{env, fs};
+
+use cached_path::{Cache, Options};
+use zip::ZipArchive;
```

Comment:
`cached_path` supports extracting archives ([ref](https://docs.rs/cached-path/latest/cached_path/struct.Options.html#method.extract)) which you already use, though only when force fetching?